config/params.php

```diff
@@ -79,6 +79,9 @@
         'optionalRequests' => [
             '/assets/*',
         ],
+        'optionalCommands' => [
```

Comment:
Would it be better to rename it?

```suggestion
        'ignoreCommands' => [
```

Reply:
The better also rename `optionalRequests`